commit 40e876a63aa0e10f378c7559f9df688a5ae4f27b
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2019-05-27T16:50:59+00:00 (4 months ago)
    
    TRITON-1049 Migration (RFD34) limit values should be stored inside SAPI

diff --git a/lib/apis/wfapi.js b/lib/apis/wfapi.js
index bbcebc4..7688bf9 100644
--- a/lib/apis/wfapi.js
+++ b/lib/apis/wfapi.js
@@ -813,6 +813,7 @@ Wfapi.prototype.createMigrateJob = function _createMigrateJobFn(req, cb) {
         override_alias: req.params.override_alias,
         server_uuid: req.vm.server_uuid, // Used by the acquireVMTicket calls.
         target: '/migrate-' + vm_uuid,
+        zfs_send_mbps_limit: req.app.options.zfs_send_mbps_limit,
         task: workflowTask,
         vm: req.vm,
         vm_uuid: vm_uuid,
diff --git a/package.json b/package.json
index 90e7416..d4ad1b3 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.8.7",
+  "version": "9.8.8",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
diff --git a/sapi_manifests/vmapi/template b/sapi_manifests/vmapi/template
index 139ea48..b7861c2 100644
--- a/sapi_manifests/vmapi/template
+++ b/sapi_manifests/vmapi/template
@@ -27,6 +27,9 @@
         "enabled": false
 {{/fabric_cfg}}
     },
+{{#zfs_send_mbps_limit}}
+    "zfs_send_mbps_limit": {{zfs_send_mbps_limit}},
+{{/zfs_send_mbps_limit}}
     "wfapi": {
         "forceMd5Check": true,
         "workflows": [
diff --git a/server.js b/server.js
index f2b367f..09cbb40 100644
--- a/server.js
+++ b/server.js
@@ -303,6 +303,7 @@ function startVmapiService() {
                 metricsManager: metricsManager,
                 moray: moray,
                 morayBucketsInitializer: morayBucketsInitializer,
+                zfs_send_mbps_limit: config.zfs_send_mbps_limit,
                 overlay: config.overlay,
                 cnapi: config.cnapi, // TODO: TRITON-1295
                 reserveKvmStorage: config.reserveKvmStorage,
